modules/botpress-nlu/src/index.js

```diff
@@ -84,6 +84,10 @@ module.exports = {
           intent: {
             is: intentName =>
               (_.get(event, 'nlu.intent.name') || '').toLowerCase() === (intentName && intentName.toLowerCase())
+          },
+          intents: {
+            has: intentName =>
+              (_.get(event, 'nlu.intents') || []).find(i => i.name === intentName) !== undefined
```

Comment:
Can use `!!` instead of ` !== undefined`, as I'm actually not sure it returns undefined but null? `!!` is more tolerant